tests/integration/test_properties.py

```diff
@@ -152,8 +152,10 @@ def test_archive_property_by_id(self, valid_header, create_property):
         test_property = create_property()
 
         """The archive property endpoint should return a 200 code when successful"""
-        responseSuccess = self.client.post(
-            f"/api/properties/archive/{test_property.id}", headers=valid_header
+        responseSuccess = self.client.put(
+            f"/api/properties/{test_property.id}",
```

Comment:
what's the f for?

Reply:
That's a good question. It's for string interpolation. Python has a few different ways to do string interpolation. This particular way is called "f-strings".

**Resources:**

- https://www.programiz.com/python-programming/string-interpolation
- https://www.python.org/dev/peps/pep-0498/